.github/workflows/deploy-wheels.yml

```diff
@@ -2,97 +2,96 @@
 name: Build sdist and wheels
 
 on:
-  push:
-  pull_request:
-  release:
-    types:
-      - published
+    push:
```

Comment:
Looks like some autoformatter had some fun with this file. Could you replace the original indentation?

Reply:
Yep. that was prettier. Indentation should be reverted now.